src/main/scala/firrtl/stage/phases/DriverCompatibility.scala

```diff
@@ -0,0 +1,102 @@
+package firrtl.stage.phases
+
+import firrtl.stage._
+
+import firrtl.{AnnotationSeq, FileUtils, Parser}
+import firrtl.proto.FromProto
+import firrtl.options.{InputAnnotationFileAnnotation, Phase, StageOptions, StageUtils}
+import firrtl.options.Viewer.view
+
+import java.io.File
+
+/** Provides compatibility methods to create old Driver behavior where command line options, if missing, are inferred
+  * based on other options.
+  */
+object DriverCompatibility {
+  /** Determine the top name using the following precedence (highest to lowest):
+    *  - Explicitly from the user-specified `--top-name`
+    *  - Implicitly from the top module ([[ir.Circuit.main]]) of [[FirrtlCircuitAnnotation]]
```

Comment:
should be firrtl.ir.Circuit for all these lines.
HashFirrtlOptions not found at lines 25 and 44

Reply:
Fixed up. I made all of `DriverCompatibility` `private [firrtl]` as this really shouldn't be exposed externally.